src/Microsoft.Framework.PackageManager/Utils/VersionUtils.cs

```diff
@@ -25,7 +26,7 @@ private static string GetActiveRuntimeName()
 
             if (!string.IsNullOrEmpty(pathVariable))
             {
-                string dnuExecutable = PlatformHelper.IsWindows ? "dnu.cmd" : "dnu";
+                string dnuExecutable = RuntimeEnvironmentHelper.IsWindows(CallContextServiceLocator.Locator.ServiceProvider) ? "dnu.cmd" : "dnu";
```

Comment:
If we can do this, why do we want to pass `IServiceProvider/IRuntimeEnvironment` around? If we have a global instance of service provider, can we just have a property `RuntimeEnvironmentHelper.IsWindows`, in which the boolean is lazily evaluated with this global service provider?